molecule/default/tests/test_create_and_assign_floating_ip.py

```diff
@@ -18,36 +19,18 @@
 floating_ip_name = None
 
 
-def create_instance_from_image(run_on_host):
-    """Create a test instance from image"""
-    image_id = utils.get_id_by_name('image', image_name, run_on_host)
-    assert image_id is not None
-
-    network_id = utils.get_id_by_name('network', private_net, run_on_host)
-    assert network_id is not None
-
-    cmd = "{} openstack server create --image {} --flavor {} --nic net-id={} {}'".format(utility_container, image_id,
-                                                                                         flavor, network_id,
-                                                                                         instance_name)
-    run_on_host.run_expect([0], cmd)
-
-    assert (utils.verify_asset_in_list('server', instance_name, run_on_host))
-    assert (utils.get_expected_value('server', instance_name, 'status', 'ACTIVE', run_on_host))
-    assert (utils.get_expected_value('server', instance_name, 'OS-EXT-STS:power_state', 'Running', run_on_host))
-
-
 @pytest.mark.test_id('a97e1202-796a-11e8-ba13-525400bd8005')
 @pytest.mark.jira('asc-254')
 @pytest.mark.run(order=1)
 def test_create_floating_ip(host):
     """Create floating IP"""
 
     global floating_ip_name
-    floating_ip_name = utils.create_floating_ip(getway_net, host)
-    assert (floating_ip_name is not None)
+    floating_ip_name = helpers.create_floating_ip(getway_net, host)
```

Comment:
Although it works with the current PR, the misspelling of s/getway_net/gateway_net/ should be corrected.

Reply:
@johnduarte updated